x/derivatives/types/keys.go

```diff
@@ -27,19 +27,16 @@ const (
 )
 
 const (
+	// TODO: KeyPrefixDerivativesSubpoolAssets is unused. Remove it if it won't be necesary.
 	// subpool assets
 	KeyPrefixDerivativesSubpoolAssets = "subpool_assets"
-	// user deposited real assets
-	KeyPrefixDerivativesUserDepositedAssets = "user_deposited_assets"
-	// User deposits by address
-	KeyPrefixPoolDeposit = "pool_deposit"
-	//
-	KeyPrefixPosition              = "position"
-	KeyPrefixUserPosition          = "user_position"
-	KeyPrefixPerpetualFutures      = "perpetual_futures"
-	KeyPrefixPerpetualOptions      = "perpetual_options"
-	KeyPrefixNetPositionAmount     = "net_position_amount"
-	KeyPrefixLastPositionId        = "last_position_id"
+	KeyPrefixPosition                 = "position"
+	KeyPrefixUserPosition             = "user_position"
+	KeyPrefixPerpetualFutures         = "perpetual_futures"
+	KeyPrefixPerpetualOptions         = "perpetual_options"
+	KeyPrefixNetPositionAmount        = "net_position_amount"
+	KeyPrefixLastPositionId           = "last_position_id"
+	// TODO: KeyPrefixAccumulatedFee is unused. Remove it if it won't be necesary.
 	KeyPrefixAccumulatedFee        = "accumulated_fee"
```

Comment:
@KimuraYu45z this item will need to be removed?

Reply:
Sorry I forgot...